angular/core/models/group_model.coffee

```diff
@@ -121,7 +120,10 @@ angular.module('loomioApp').factory 'GroupModel', (DraftableModel, AppConfig) ->
       @groupPrivacy == 'secret'
 
     allowPublicDiscussions: ->
-      @discussionPrivacyOptions != 'private_only'
+      if @isNew()
```

Comment:
Maybe this is a naming thing but I'm a little confused here; what happens when you make a new group and set it to 'secret'? Then allowPublicDiscussions would want to be false?